ciri/consensus/tangle/tangle.h

```diff
@@ -107,6 +107,8 @@ retcode_t iota_tangle_transaction_approvers_count(tangle_t const *const tangle,
 
 retcode_t iota_tangle_transaction_metadata_clear(tangle_t const *const tangle);
 
+retcode_t iota_tangle_transaction_delete_transactions(tangle_t const *const tangle, hash243_set_t const hashes);
```

Comment:
Can you make it `iota_tangle_transactions_delete` for consistency ?

Reply:
@tsvisabo

Reply:
i would think it is less consistent like that, `iota_tangle_transaction_` is the prefix

Reply:
But we already have some plural suffix `iota_tangle_transactions_` for sets, like `iota_tangle_transactions_update_snapshot_index` or `iota_tangle_transactions_update_solid_state`

Reply:
ok ok

Reply:
i agree it's nicer like that actually